### PR TITLE
Add instructions for openSUSE 42.3

### PR DESCRIPTION
**Pull request by mattiasgiese**
This pull request adds support for building openSUSE 42.3 images.

### CODE REVIEW

**Comment by rickard-von-essen** [comment]
Which builders did you test this with?

**Comment by mattiasgiese** [comment]
I could only test with virtualbox-iso for now, will try to build with libvirt/qemu today.

**Comment by tas50** [comment]
The existing 42.2 files can go away and we should probably make the folder path for the auto install file just 42 since there's no need to rename it every time openSUSE releases a point release.

File: http/opensuse-leap-42.3/autoinst.xml [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+  </bootloader>
+
+  <networking>
+    <ipv6 config:type="boolean">false</ipv6>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+    <dns>
+      <dhcp_hostname config:type="boolean">true</dhcp_hostname>
+      <dhcp_resolv config:type="boolean">true</dhcp_resolv>
+      <domain>vagrantup.com</domain>
+      <hostname>opensuse-leap-42.3-x64</hostname>
```
Comment (rickard-von-essen) [inline http/opensuse-leap-42.3/autoinst.xml]: Unfortunately we have the minior version in the hostname here.

Maybe we can provide the hostname in the `boot_command` as a [parameter](https://doc.opensuse.org/projects/autoyast/#idm140518702500384) to `autoinst.xml`.

Comment (tas50) [inline http/opensuse-leap-42.3/autoinst.xml]: Even if we use the 42 for the hostname it will be correct since that's the major.

Comment (mattiasgiese) [inline http/opensuse-leap-42.3/autoinst.xml]: Yep, that works. I will make these changes and open another pull request

**Comment by mattiasgiese** [comment]
I think i'll try to refactor this one a bit because the added scripts are merely workarounds.

**Comment by tas50** [comment]
I've opened #869 to replace this. Thanks for the initial push

**Pull request closed without merging**